src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -74,7 +74,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     if (!userStore.isUserRegistered(username)) {
-      User user = new User(UUID.randomUUID(), username, Instant.now());
+      // TODO: add password
```

Comment:
would recommend the team complete review of code and merge these changes as they have impact on other assignments.